<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(173, 194, 169)</string>
  </property>
  <widget class="QLabel" name="closemouth">
   <property name="geometry">
    <rect>
     <x>585</x>
     <y>30</y>
     <width>751</width>
     <height>900</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(201, 232, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>방송부 버츄얼 그림/일곽버츄얼호랑이/A3.PNG</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="returnbutton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한컴 고딕</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(211, 228, 205)</string>
   </property>
   <property name="text">
    <string>뒤로가기</string>
   </property>
  </widget>
  <widget class="QLabel" name="speakinggif">
   <property name="geometry">
    <rect>
     <x>585</x>
     <y>30</y>
     <width>751</width>
     <height>901</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(201, 211, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="tosheetbutton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>150</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한컴 고딕</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(211, 228, 205)</string>
   </property>
   <property name="text">
    <string>시트로 이동</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getsheetbutton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>880</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한컴 고딕</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(211, 228, 205)</string>
   </property>
   <property name="text">
    <string>불러오기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="launchbutton">
   <property name="geometry">
    <rect>
     <x>1360</x>
     <y>880</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한컴 고딕</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(211, 228, 205)</string>
   </property>
   <property name="text">
    <string>실행</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>950</y>
     <width>1141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(254, 245, 237)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitbutton">
   <property name="geometry">
    <rect>
     <x>1360</x>
     <y>30</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한컴 고딕</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(153, 167, 153)</string>
   </property>
   <property name="text">
    <string>종료</string>
   </property>
  </widget>
  <widget class="QPushButton" name="settingbutton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한컴 고딕</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(211, 228, 205)</string>
   </property>
   <property name="text">
    <string>설정</string>
   </property>
  </widget>
  <zorder>speakinggif</zorder>
  <zorder>closemouth</zorder>
  <zorder>returnbutton</zorder>
  <zorder>tosheetbutton</zorder>
  <zorder>getsheetbutton</zorder>
  <zorder>launchbutton</zorder>
  <zorder>comboBox</zorder>
  <zorder>quitbutton</zorder>
  <zorder>settingbutton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
